grammars/keywords-nb_NO.lark

```diff
@@ -0,0 +1,30 @@
+_PRINT: "skriv" | "print"
+_ASK: "spør" | "ask"
+_ECHO: "ekko" | "echo"
+_FORWARD: "forover" | "forward"
+_TURN: "snu" | "turn"
+right: "høyre" | "right"
+left: "venstre" | "left"
```

Comment:
should these be `_RIGHT` and `_LEFT`?

Reply:
Nope! As these are tokens instead of keys they should be lowercase. (correct me if I'm wrong @Felienne)

Reply:
You are right! It is linke `random` (also a token)